corehq/apps/builds/jadjar.py

```diff
@@ -132,7 +134,7 @@ def pack(self, files, jad_properties=None):
         buffer = StringIO(self.jar)
         with ZipFile(buffer, 'a', ZIP_DEFLATED) as zipper:
             for path in files:
-                zipper.writestr(path, files[path])
+                zipper.writestr(path, files[path].replace(BASE_ADDRESS, J2ME_ADDRESS))
```

Comment:
`files[path]` is actually a string (xml) not a path to a file. Although just thought of a better way to do this, so I'll rework it.